packages/onefetch/build.sh

```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/o2sh/onefetch
+TERMUX_PKG_DESCRIPTION="A command-line Git information tool written in Rust"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@ELWAER-M"
+TERMUX_PKG_VERSION=2.11.0
+TERMUX_PKG_SRCURL=https://github.com/o2sh/onefetch/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=ffd3cc3bd24e299ede1fada2b2da8bf066d59219da167477e1997c860650c192
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_make() {
+    termux_setup_rust
```

Comment:
Please use tabs for indentation in Shell scripts, as per <a href="https://github.com/termux/termux-packages/wiki/Coding-guideline">Coding guideline</a>.